flybywiresim/aircraft
# feat(a380x/mfd): basic FIX INFO functionality and lat rev menu fixes

## Summary of Changes


- [x] Basic FIX INFO functionality
  - [x] NOT IN DATABASE
  - [x] Placeholders for predictions text
  - [x] Misisng details (ABEAM button, etc.)
  - [x] Make sure alll true/mag stuff is correct
- [x] Fixes some instances of lateral revisions being available on flight plan elements where they should not be

## Testing instructions


- Insert FIX INFOs for different points
  - Waypoints
  - VORs/NDBs
  - Airports
  - Runways (format: `KSFO28R`)
- Make sure FIX INFO data remains stable when switching away/to the page, switching tabs, between both MFDs, etc.
- Make sure testing instructions from https://github.com/flybywiresim/aircraft/pull/8182 still pass
- Make sure A320 FIX INFO functionality is still working correctly


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
